escrow/src/main.sw

```diff
@@ -138,11 +145,8 @@ impl Escrow for Contract {
         // TODO: https://github.com/FuelLabs/sway/issues/2014
         //       `.contains() -> bool / .position() -> u64` would clean up the loop
```

Comment:
Is this comment still valid since you're using the `.get()` function below? Not entirely sure what this is referring to here.

Reply:
Yes, the idea was to say "is `msg_asset_id()` in here?". This way the loop is abstracted into the vec and I can remove the loop in the contract itself.